Libraries/BlePhy/MAX32690/libphy.log

```diff
@@ -115,4 +115,4 @@ arm-none-eabi-gcc-ar rcs -o ../../libphy.a obj/pal_bb.o obj/pal_bb_ble.o obj/pal
 arm-none-eabi-strip --strip-unneeded ../../libphy.a
 +++ Copying: ../../libphy.a to msdk
 cp ../../libphy.a ../../../../../Libraries/BlePhy/MAX32690/libphy.a
-revision: 52e307e
+revision: d3f6d97
```

Comment:
There was a bug in the previous version where we would have a pending interrupt after completing the deep sleep recovery. The fix is to clear the interrupts after the re-initialization.